app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -364,7 +366,8 @@ Mautic.closeBuilder = function(model) {
             // Clear the customize forms
             mQuery('#slot-form-container, #section-form-container').html('');
 
-            customHtml = '<!DOCTYPE html>'+themeHtml.find('html').get(0).outerHTML
+            customHtml = themeHtml.find('html').get(0).outerHTML;
+            customHtml = xs.serializeToString(doctype) + customHtml;
```

Comment:
Hey @escopecz, so I made some changes so that we fetch the actual doctype from the theme using an `XMLSerializer` which seems compatible enough (https://caniuse.com/#feat=xml-serializer).

Actually, using this, I don't think we even need to `find('html')` first. I think we could directly say `customHtml = xs.serializeToString(themeHtml.get(0))` unless there is something I'm overlooking.